src/cli/annotations.rs

```diff
@@ -31,8 +31,11 @@ impl Execute for Stats {
     where
         W: WriteColor,
     {
-        let document = Document::load(&self.file)
-            .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;
+
+        let document = Document::load(self.file)  
+        
+        .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;  
+
```

Comment:
```suggestion
        let document = Document::load(&self.file)  
            .with_context(|| format!("Failed to read PDF from: {:?}.", self.file))?;  
```

This is the reason this was breaking (the removal of `&`).